test/hotspot/jtreg/runtime/CommandLine/OptionsValidation/TestOptionsWithRanges.java

```diff
@@ -257,6 +257,13 @@ public static void main(String[] args) throws Exception {
         excludeTestMaxRange("NonNMethodCodeHeapSize");
         excludeTestMaxRange("CodeCacheExpansionSize");
 
+        /*
+         * Exclude CompileThresholdScaling from max range testing, because
+         * it is expected to print "outside the allowed range" warnings for
+         * the scaled flags
+         */
```

Comment:
This comment is not very clear. Do you mean

```
Max range tests expect a warning message of "outside the allowed range"
but CompileThresholdScaling prints a different warning message when it's
set too high.
```

Reply:
I changed to comment. I hope it is clearer now